README.md

```diff
@@ -553,22 +553,33 @@ interface LayoutImage extends Node {
 
 - **LayoutImage** is a workaround to handle pre-existing articles that were published using `<img>` tags rather than `<ft-content>` images. The reason for this was that in the bodyXML, layout nodes were inside an `<experimental>` tag, and that didn't support publishing `<ft-content>`.
 
-### TODO: `Table`
+### `Table`
 
 ```ts
+type TableColumnSettings = {
+	hideOnMobile: boolean
+	sortable: boolean
+	sortType: "text" | "number" | "date" | "currency" | "percent"
+}
+
 interface Table extends Parent {
 	type: "table"
-	children: [Caption | TableHead | TableBody]
+	title: string
+	footer: string
```

Comment:
**question**: the footer can contain formatting, so does this need to be `Phrasing[]` instead of `string`? or maybe its own node?

Reply:
ah, that's a shame. i didn't realize the footer could contain formatting. If we need formatting in the footer then `TableFooter` should be a Node and therefore a child. Which will maybe mean rethinking the `children` of table not to just be a list of cells. idk, maybe it's fine... `children: [...TableCell[], TableFooter?]`

Reply:
kind of gross though, idk. feel free to rethink table if you like and commit to this branch.

Nodes are always kept in `children`, in every other definition in `content-tree`. But there's actually no reason for that other than it just seemed like the right thing to do in every case so far

Reply:
i'm going to experiment with that structure in my tables branch